LAYOUT_STYLE_GUIDE.md

```diff
@@ -378,6 +379,31 @@ Will result in the following output:
 
 - Next bullet.
 
+## Note boxes
+
+Note boxes can be added by wrapping the content in a `div` with the class `lesson-note`. This will add styling to make the note stand out visually to users.
+
+A heading can be added to a note by using an `h4` element. When adding a heading, be sure to provide text that helps describe the note rather than "A note" or "Warning".
+
+### Variations
+
+Note boxes come in two variations, which can be set by adding an extra class together with `lesson-note`:
+- `lesson-note--tip` for tips or general information
+- `lesson-note--warning` for warnings about potential issues/pitfalls, and are more severe than a tip
+
+### Example
+~~~markdown
+<div class="lesson-note">
+<h4>An optional title</h4>
+A sample note box.
+</div>
+~~~
+~~~markdown
+<div class="lesson-note lesson-note--tip">
+<h4>An optional title</h4>
+A sample note box, variation: tip.
+</div>
+~~~
```

Comment:
```suggestion
~~~

```
 
nit: There needs to be a newline here for the markdown to be reliable outside GH :)